Tests/SwiftSyntaxBuilderTest/TriviaTests.swift

```diff
@@ -83,12 +83,14 @@ final class TriviaTests: XCTestCase {
           AttributeSyntax(attributeName: TypeSyntax("A")).with(\.trailingTrivia, .space)
           AttributeSyntax(attributeName: TypeSyntax("B")).with(\.trailingTrivia, .space)
         },
-        "@A @B "
+        """
+        @A @B 
```

Comment:
I would prefer this string literal to continue being single line so you can actually see the trailing whitespace.